### DISCUSSION TITLE
Feature request: define custom label for self-hosted runner

### DISCUSSION THREAD

**Discussion by julien-c** [community/community#26176] [Actions]
<p>I have multiple self-hosted runners that are all running the same OS, so I can’t filter them using the OS type.</p>
<p>I’d like to be able to select them using a custom label, or at least the self-hosted runner’s name (as defined/displayed in the repo’s settings/actions page)</p>
<p><a href="https://github.community/t/can-workflow-specify-which-self-runner-to-run-a-job/17405">This post</a> by <a class="mention" href="/yanjingzhu">@yanjingzhu</a> mentions that this is not yet supported, so I’d like to strongly +1 this feature.</p>

**Comment by MChevy422** [comment]
<p>Hello <span><a class="mention" href="/julien-c">@julien-c</a></span>, <br><br>Thank you for your feedback! We’re always working to improve GitHub and the GitHub Community Forum, and we consider every suggestion we receive. I’ve logged your feature request in our internal feature request list. Though I can’t guarantee anything or share a timeline for this, I can tell you that it’s been shared with the appropriate teams for consideration.<br><br>Once again, thank you for your input!<br><br>Greatly appreciated<br><br>MChevy422</p>

**Comment by Yanjingzhu** [comment, marked as answer]
<p>Github engineering team are going to support org-level self hosted runners and custom labels by 31 March 2020.</p>
<p>Let’s wait for the feature to be released. You could keep an eye on <a href="https://github.blog/changelog/" rel="nofollow noopener">Changelog</a> site for the new features of Github Actions. </p>
